playground/mocks/data/idp/idx/user-unlock-account.json

```diff
@@ -1,177 +1,167 @@
 {
-  "stateHandle": "02bgjl5UBobbixr5M3Qn6BI9JwRyhjD1It2rzy3zzZ",
-  "version": "1.0.0",
-  "expiresAt": "2021-01-29T08:45:30.000Z",
-  "intent": "LOGIN",
-  "remediation": {
-      "type": "array",
-      "value": [
+  "version":"1.0.0",
+  "stateHandle":"02bgjl5UBobbixr5M3Qn6BI9JwRyhjD1It2rzy3zzZ",
+  "expiresAt":"2021-05-07T22:17:33.000Z",
+  "intent":"LOGIN",
+  "remediation":{
+    "type":"array",
+    "value":[
+      {
+        "rel":[
+          "create-form"
+        ],
+        "name":"select-authenticator-unlock-account",
+        "href":"http://localhost:3000/idp/idx/challenge",
+        "method":"POST",
+        "produces":"application/ion+json; okta-version=1.0.0",
+        "value":[
           {
-              "rel": [
-                  "create-form"
-              ],
-              "name": "select-authenticator-unlock-account",
-              "href": "http://localhost:3000/idp/idx/challenge",
-              "method": "POST",
-              "produces": "application/ion+json; okta-version=1.0.0",
-              "value": [
-                  {
-                      "name": "authenticator",
-                      "type": "object",
-                      "options": [
-                          {
-                              "label": "Email",
-                              "value": {
-                                  "form": {
-                                      "value": [
-                                          {
-                                              "name": "id",
-                                              "required": true,
-                                              "value": "aut215n1MxHoGZdfd0g4",
-                                              "mutable": false
-                                          },
-                                          {
-                                              "name": "methodType",
-                                              "required": false,
-                                              "value": "email",
-                                              "mutable": false
-                                          }
-                                      ]
-                                  }
-                              },
-                              "relatesTo": "$.authenticatorEnrollments.value[0]"
-                          },
+            "name":"identifier",
+            "label":"Username",
```

Comment:
This is the main reason for updating this mock. This label used to be `Email or Username`.